### PR TITLE
Continuation of ride restriction

### PR DESCRIPTION
**Pull request by smartens21**
Api Work done.
Data seeder join attractions and riderestrictions don't work. Help!

### CODE REVIEW

**Comment by M4Al** [comment]
Fixex that for you. Please approve the PR if you think it's fine

File: src/PP.Domain/Attractions/Attraction.cs [hunk 1]
```diff
@@ -12,5 +12,11 @@ public class Attraction : AuditedAggregateRoot<Guid>
 
         public ICollection<RideRestriction> RideRestrictions { get; set; }
 
+        public Attraction()
```
Comment (M4Al) [inline src/PP.Domain/Attractions/Attraction.cs:15]: I added a constructor here to initialise the list to an empty List. This seems a bit fishy to me. There must be a better way, but we'll take this solution for now.

File: src/PP.Domain/IDataSeedContributor.cs [hunk 2]
```diff
@@ -10,64 +11,71 @@ namespace PP
...
+            var begeleiding = await _rideRestrictionRepository.InsertAsync(new RideRestriction
+            {
+                Name = "90 cm - 1m20 begeleiding",
+                Type = RideRestrictionType.Supervision,
+                ValueLow = 900,
+                ValueHigh = 1200
+            }
+            );
+
+            elRio.RideRestrictions.Add(Volwassen12m);
```
Comment (M4Al) [inline src/PP.Domain/IDataSeedContributor.cs:73]: SImple as it should be

File: src/PP.Domain/RideRestrictions/RideRestriction.cs [hunk 3]
```diff
@@ -13,5 +13,10 @@ public class RideRestriction :AuditedAggregateRoot<Guid>
         public int ValueHigh { get; set; }
 
         public ICollection<Attraction> Attractions { get; set; }
+
+        public RideRestriction()
```
Comment (M4Al) [inline src/PP.Domain/RideRestrictions/RideRestriction.cs:17]: Same thing as with the Attractions

**Review by M4Al — APPROVED** [review]

File: src/PP.Domain/IDataSeedContributor.cs (hunk 2 above)
Comment (smartens21) [reply src/PP.Domain/IDataSeedContributor.cs:73]: Ik wist dat het iets simpel ging zijn. Maar hier had ik niet aan gedacht.

**Pull request merged**